frontmatter/introduction.markdown

```diff
@@ -20,16 +20,85 @@ This book is split into seventeen main sections:
 
 ### Section I: Section Name Here
 
-A paragraph or two about this section. Replicate exactly what you put in the section-intro-n.markdown file.
+A paragraph or two about this section. Replicate exactly what you put in the section-intro-1.markdown file.
 
 ### Section II: Section Name Here
 
-A paragraph or two about this section. Replicate exactly what you put in the section-intro-n.markdown file.
+A paragraph or two about this section. Replicate exactly what you put in the section-intro-2.markdown file.
 
 ### Section N: Section Name Here
 
 A paragraph or two about this section. Replicate exactly what you put in the section-intro-n.markdown file.
 
+# Example - Jetpack Compose Book
```

Comment:
Hey, Is this meant to stay in the actual frontmatter? Can this go in the samples folder as well, please

Reply:
I'll move it to the samples! Sorry for missing that one!